src/includes/class-wordlift.php

```diff
@@ -614,6 +623,7 @@ private function load_dependencies() {
 
 		//** WordPress Admin */
 		$this->download_your_data_page = new Wordlift_Admin_Download_Your_Data_Page();
+		$this->install_wizard = new Wordlift_Install_wizard();
```

Comment:
We try to stick to WP PHP Coding Standards ([indentation](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation)).